osgtest/tests/test_838_xrootd_tpc.py

```diff
@@ -5,12 +5,16 @@
 
 class TestStopXrootdTPC(osgunittest.OSGTestCase):
     @core.elrelease(7,8)
+    def setUp(self):
+        core.skip_ok_unless_installed("xrootd",
+                                      by_dependency=True)
+
     def test_01_stop_xrootd(self):
         if core.state['xrootd.tpc.backups-exist']:
             files.restore(core.config['xrootd.tpc.config-1'], "xrootd")
             files.restore(core.config['xrootd.tpc.config-2'], "xrootd")
 
-        core.skip_ok_unless_installed('xrootd', 'xrootd-scitokens', by_dependency=True)
+        core.skip_ok_unless_installed('xrootd-scitokens', by_dependency=True)
```

Comment:
This should only depend on xrootd but since you did it before the dependency on scitokens should be removed.